background/score/activities.go

```diff
@@ -162,6 +163,12 @@ func (s *ScoreUpdateWorker) RefreshLocationStateActivity(ctx context.Context, ac
 			return nil, err
 		}
 
+		if time.Since(profile.LastNudge[schema.BehaviorOnSymptomSpikeNudge]) > 90*time.Minute { // 90 minutes of delay between nudges
+			if profile.Metric.SymptomDelta <= 110 && metric.SymptomDelta > 110 { // from a non-spike area to a spike area
```

Comment:
I thought it's `metric.Details.Symptoms.Score / profile.Metric.Details.Symptoms.Score > 1.1`.

Reply:
I think it is the same. I checked the calculation. Unless we are going to remove that field. Even we are going to remove that field I will still have another computed field for displaying on the client. I will then use that field directly.

Reply:
I think the statement `metric.Details.Symptoms.Score / profile.Metric.Details.Symptoms.Score > 1.1` means the spike of symptom score's delta.

But for the statement  symptom spike means the `SymptomDelta > 110`

Reply:
https://github.com/bitmark-inc/autonomy-api/blob/master/score/symptom.go#L76

Reply:
?

Reply:
Note:
new/old > 1.1
new/old - 1 > 0.1
(new - old)/old > 0.1
`SymptomDelta` captures the growth rate, so the criteria should be 10% (0.1).